V5/SpinUp/src/main.cpp

```diff
@@ -21,10 +21,101 @@ void pre_auton( void ) {
 }
 
 void autonomous( void ) {
-
+  
 }
 
 void usercontrol( void ) {
+  double driveMode = 0;
```

Comment:
@coco131313  what is drivemode? why do we need it and does your code actually use it somewhere?